test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -31,4 +31,9 @@
   <ItemGroup>
     <None Include="Scenarios\Api\AspirationalExamples.cs" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <PackageReference Include="Microsft.ML.TensorFlow.TestModels" Version="0.0.1-test" />
```

Comment:
>Microsft.ML.TensorFlow.TestModels [](start = 31, length = 33)

Is this package coming from myget ?

Reply:
Yes, I pushed it to [https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsft.ML.TensorFlow.TestModels](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsft.ML.TensorFlow.TestModels)

---
In reply to: [212693208](https://github.com/dotnet/machinelearning/pull/704#discussion_r212693208) [](ancestors = 212693208)